platform/pulpcore/app/viewsets/repository.py

```diff
@@ -51,6 +51,20 @@ def importers(self, request, name):
         serializer = ImporterSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
 
+
+    def update(self, request, *args, **kwargs):
```

Comment:
I took my function signature cues from https://github.com/pulp/pulp/pull/3046/files and https://github.com/pulp/pulp/pull/3048/files 
I like having a consistent update function signature, but since neither of those are merged yet, I guess I can set the example :)